sdk/version/version_base.go

```diff
@@ -9,6 +9,6 @@ var (
 	CgoEnabled bool
 
 	Version           = "1.10.0"
-	VersionPrerelease = "dev"
+	VersionPrerelease = "dev-1"
```

Comment:
```suggestion
	VersionPrerelease = "dev1"
```
To be in keeping with the RC format.